src/main/java/com/google/api/gax/grpc/InstantiatingChannelProvider.java

```diff
@@ -30,6 +30,7 @@
 package com.google.api.gax.grpc;
 
 import com.google.api.gax.core.CredentialsProvider;
+import com.google.api.gax.utils.PropertiesProvider;
```

Comment:
I don't think it's necessary to create a new package for this; I think would fit just fine in gax.core.